connectors/filtering/basic_rule.py

```diff
@@ -147,6 +147,50 @@ def from_string(cls, string):
                 )
 
 
+class BasicRuleValidationResult:
```

Comment:
No need to review (temporary copy from https://github.com/elastic/connectors-python/pull/268)